src/openrct2/paint/tile_element/Paint.Surface.cpp

```diff
@@ -1104,81 +999,30 @@ void surface_paint(paint_session* session, uint8_t direction, uint16_t height, c
     {
         const bool showGridlines = (session->ViewFlags & VIEWPORT_FLAG_GRIDLINES);
 
-        int32_t branch = -1;
-        if (tileElement->AsSurface()->GetSurfaceStyle() == TERRAIN_GRASS)
```

Comment:
Rather than removing this `if` statement, wouldn't it be better to check for the `canGrow` property?